pymesync.py

```diff
@@ -247,3 +275,30 @@ def _format_endpoints(self, queries):
             query_string += "?{}".format("&".join(query_list))
 
         return query_string
+
+    def _get_field_errors(self, actual, object_name):
+        """Checks that ``actual`` parameter passed to POST method contains
+        items in required or optional lists for that ``object_name``.
+        Returns None if no errors found or error string if error found"""
+        missing_list = list(self.required_params[object_name])
```

Comment:
This is a very minor nitpick, but you don't need to bother assigning this variable until you are sure the next test passes - check that you have a dict first, then do everything else.